analyzer/windows/modules/packages/exe.py

```diff
@@ -11,6 +11,28 @@
 class Exe(Package):
     """EXE analysis package."""
 
+    REGKEYS = [
+        [
+            HKEY_LOCAL_MACHINE,
+            "Software\\Microsoft\\Security Center",
+            {
+                # "Would you like Internet Explorer as default browser?"
```

Comment:
the point here with IE stuff is to lower the noise when you run (for example) ransomware binaries (not droppers) which bring up IE in the end or during analysis. As mentioned in the PR - in long term it makes much more sense to flip code around and build profiles for vm-s rather than inject registry changes in packages.